app/app.js

```diff
@@ -1232,11 +1232,42 @@ var AppComponent = React.createClass({
       return null;
     }
 
+    var mostRecentUpload = _.sortBy(_.where(data, {type: 'upload'}), function(d) {
+      return Date.parse(d.time);
+    }).reverse()[0];
+    var timePrefsForTideline;
+    if (!_.isEmpty(mostRecentUpload) && !_.isEmpty(mostRecentUpload.timezone)) {
+      try {
+        sundial.checkTimezoneName(mostRecentUpload.timezone);
+        timePrefsForTideline = {
+          timezoneAware: true,
+          timezoneName: mostRecentUpload.timezone
+        };
+      }
+      catch(err) {
+        app.log(err);
```

Comment:
If you don't have a timezone from your data and there isn't one in query params, we default to the non-timezone aware delay (the current prod way of doing things). That's [here](https://github.com/tidepool-org/blip/blob/jebeck/tz-for-utc-bootstrap/app/app.js#L146) in the code. I did just remove the old hard-coded default timezoneName and change that to `null`.